test/WebJobs.Script.Tests.Integration/Host/StandbyManagerTests.cs

```diff
@@ -156,5 +154,10 @@ public void IsWarmUpRequest_ReturnsExpectedValue()
                 WebScriptHostManager.ResetStandbyMode();
             }
         }
+
+        public void Dispose()
+        {
+            ScriptSettingsManager.Instance.Reset();
```

Comment:
@fabiocav -- this was ultimately the issue that I was having yesterday. This test was enabling StandbyMode but never resetting it. Then, future tests would try to start a host in StandbyMode and everything would blow up. That's where those 50 failures I mentioned to you were coming from. Now they're all good.

Reply:
Yes, there are other subtle issues with the way the configuration is passed in during the tests (a reset in that case will end up calling the factory that,  in turn, returns the same configuration instance, and since the default environment variable configuration provider caches the environment variable values after it is initialized, changes are not correctly picked up. @mathewc created this to track: https://github.com/Azure/azure-functions-host/issues/2526 . I'll work on that soon.

Reply:
Exactly. I had to step through to figure out what was going on. The Environment looked right, but the Instance was never updated. Using a static instance doesn't help either.